client.go

```diff
@@ -67,8 +67,9 @@ func (c *SeabirdClient) stockCallback(event *pb.CommandEvent) {
 		} else {
 			company = fmt.Sprintf("%s", ticker)
 		}
+		percentChange := ((quote.O - quote.C) / quote.O) * 100
 		// TODO: Don't hardcoded USD here - currency requires premium https://finnhub.io/docs/api#company-profile
-		c.Replyf(event.Source, "%s: %s - Open: $%.2f, Current: $%.2f", event.Source.GetUser().GetDisplayName(), company, quote.O, quote.C)
+		c.Replyf(event.Source, "%s: %s - Open: $%.2f, Current: $%.2f (%.2f %%)", event.Source.GetUser().GetDisplayName(), company, quote.O, quote.C, percentChange)
```

Comment:
This looks good, but do we want do make sure there's a sign? Like `+42.0%` as well as `-42%`?

Reply:
Oh, I thought that's what the `%%` would do @belak

Reply:
I was talking about making sure there was a sign, I think the `+` flag does that.

https://play.golang.org/p/CB4mXAydFdY

Reply:
Ah gotcha, done!